cfme/common/__init__.py

```diff
@@ -102,13 +102,6 @@ class Validatable(object):
     """
     property_tuples = []
 
-    def properties_ui(self):
```

Comment:
@jkrocil removed this `SummarPage`. using new implementation `SummaryMixin`by @mfalesni